lyse/__init__.py

```diff
@@ -407,6 +407,22 @@ def get_trace(self,name):
             trace = h5_file['data']['traces'][name]
             return array(trace['t'],dtype=float),array(trace['values'],dtype=float)         
 
+    def get_waits(self):
+        """Return the time out status of waits.
+
+        Raises:
+            Exception: If the experiment has no waits.
+
+        Returns:
+            :obj:`numpy:numpy.ndarray`: Returns 2-D timetrace of times `'time'`
+            and time out status `'timed_out'`.
+        """
+        with h5py.File(self.h5_path,'r') as h5_file:
+            if not 'waits' in h5_file['data']:
+                raise Exception('The shot has no waits')
+            trace=h5_file['data']['waits']
```

Comment:
could we rename this variable from `trace` to `waits`, so it is more descriptive?

Reply:
Great suggestion, in the end I used much simpler solution returning all of the parameters in structured array and omitted the variable all together